src/types/Signer.ts

```diff
@@ -0,0 +1,4 @@
+interface Signer {
```

Comment:
We are going to need to export this type from `index.ts` so that client applications can easily make use of it.